interfaces/backends/backends.go

```diff
@@ -41,11 +42,16 @@ func All() []interfaces.SecurityBackend {
 		&systemd.Backend{},
 		&seccomp.Backend{},
 		&dbus.Backend{},
-		&udev.Backend{},
-		&mount.Backend{},
+	}
+
+	if !systemd_tools.IsContainer() {
+		all = append(all, &udev.Backend{})
+	}
+
```

Comment:
The multiple calls to `append(all, ...)` here feel like something that one day someone will look at and decide to just optimise away by putting the mount, kmod and polkit backends in the declaration of `all` above - if it is important to have the udev backend come before these other 3 then I think it would be worth adding a comment here stating that. Otherwise if the order is not important then it is probably worth doing this optimisation now I think and only having the single conditional call to `append()` for the udev backend.

Reply:
The order is significant. I will adjust this to explain. Thank you for the review!

Reply:
Done now

Reply:
thanks